Makefile

```diff
@@ -0,0 +1,14 @@
+MAKEFILES_VERSION=6.0.3
```

Comment:
Use 6.1.0

Reply:
Updated to 6.2.0. But we need another makefiles bugfix

Reply:
Updated to 6.3.0